src/common/MainWrapper/index.js

```diff
@@ -1,6 +1,6 @@
 import styled from "styled-components";
 
-const MainWrapper = styled.main`
+const MainWrapper = styled.div`
```

Comment:
better for semantics is main than div

Reply:
MainWrapper we should added as other-common component and change it in MovieList and PeopleList